perfkitbenchmarker/benchmarks/speccpu2006_benchmark.py

```diff
@@ -219,6 +221,8 @@ def Run(benchmark_spec):
   vm = vms[0]
   logging.info('SpecCPU2006 running on %s', vm)
   num_cpus = vm.num_cpus
+  vm.RemoteCommand("cp {0}/config/Example-simple.cfg "
```

Comment:
Oh sorry, this is small omission for my site - i have flagged this to remove (my version of cpu2006 have few troubles with compiling).